lib/hanami/view/escape.rb

```diff
@@ -165,10 +165,13 @@ def self.extended(base)
       # @since 0.4.0
       # @api private
       def method_added(method_name)
+        visibility = :private if private_method_defined? method_name
+        visibility ||= :protected if protected_method_defined? method_name
```

Comment:
I could change that to an `if .. elsif ... end` which I find particularly ugly or to a `case`.

There isn't a way that a method with the same name is at the same time `private` and `protected` and `public`.

I could make it look like this:

```ruby
visibility = :public
visibility = :private if private_method_defined? method_name
visibility = :protected if protected_method_defined? method_name
```
Does that make it clear enough? (I like this lil' a bit more).